components/Common/ThemeToggle/index.tsx

```diff
@@ -0,0 +1,40 @@
+import { MoonIcon, SunIcon } from '@heroicons/react/24/outline';
+import { AccessibleIcon } from '@radix-ui/react-accessible-icon';
+import type { FC } from 'react';
+import { useIntl } from 'react-intl';
+
+import styles from './index.module.css';
+
+type ThemeToggleProps = {
+  onChange: () => void;
+};
+
+const ThemeToggle: FC<ThemeToggleProps> = ({ onChange }) => {
+  const intl = useIntl();
+
+  return (
+    <button
+      type="button"
+      onClick={onChange}
+      onKeyDown={onChange}
+      className={styles.themeToggle}
+    >
+      <AccessibleIcon
```

Comment:
@bmuenzenmeyer just double checking, is `AccessibleIcon` needed for Buttons? (Just double-checking)...

Should we even use `onKeyDown`?

Reply:
perhaps not! I am defintely interested in their use in the footer icons however